src/main/views/user-option.njk

```diff
@@ -13,10 +14,11 @@
 {% block content %}
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
+      {{ errorSummary(error) }}
```

Comment:
I didn't add the error summary before I felt the page is very simple with only one option, and so the error summary is kind of redundant and repeating what has been said in the error message section. Do you think an error summary is good on the page?

Reply:
According to GDS, error summary must be included on all pages, even if small or a single error

Reply:
Heehee can't agree with GDS for this case but they are the boss in UI design :)

Reply:
hah me neither, I wasn't a fan but I thought might as well enforce it in case we ever do open the service to citizens